### PR TITLE
Fix length_in_seconds on lessons

### PR DESCRIPTION
**Pull request by roxanariza**
[BE-15](https://musora.atlassian.net/jira/software/c/projects/BE/boards/41?assignee=712020%3A61fc08e8-0039-472f-8851-3a58c29af95d&selectedIssue=BE-15)

Update both filters and content cards to use the same length_in_seconds field

### CODE REVIEW

File: src/Services/RailcontentV2DataSyncingService.php [hunk 1]
```diff
@@ -222,6 +215,12 @@ public function syncContentIds(array $contentIds, $forUserId = null)
             $contentColumnsToUpdate['like_count'] = $contentsLikeCounts[$contentRow->id]->count ?? 0;
 
             // length in seconds
+            $contentLengthInSecondsField = $databaseConnection
+                ->table(config('railcontent.table_prefix').'content_fields')
+                ->whereIn('content_id', [$contentColumnsToUpdate['video']])
```
Comment (BrandonKerr) [inline src/Services/RailcontentV2DataSyncingService.php:220]: Is it possible for `video` to be an undefined key in `$contentColumnsToUpdate`?

**Review by BrandonKerr — COMMENTED** [review]
I just had the one question about a possible SQL error if there isn't a video entry in the array. Other than that, it looks good!

File: src/Services/RailcontentV2DataSyncingService.php (hunk 1 above)
Comment (roxanariza) [reply src/Services/RailcontentV2DataSyncingService.php:220]: No, the video key exists always with null value by default (https://github.com/railroadmedia/railcontent/pull/66/files/d33eb7bd0eb6e67529dfceba77a30ed593a335b5#diff-6fd3f4d6fa730f6f104823c5d350105b19c68408434abe7915fe4a11bfe8d9ceR169)

Comment (BrandonKerr) [reply src/Services/RailcontentV2DataSyncingService.php:220]: Oh perfect, thank you!

**Review by BrandonKerr — APPROVED** [review]

**Pull request merged**